tests/vinyldns_tests.bats

```diff
@@ -49,6 +49,18 @@ load test_helper
   $ew --output=json group --name "ok-group" | grep "${fixture}"
 }
 
+@test "group-update (when the group exists)" {
+  fixture="$(cat tests/fixtures/group_updated)"
+  ok_group=$($ew --op json group --name "ok-group")
+  updated_group="$(echo ${ok_group} | sed 's/test@test.com/update@update.com/g')"
+  run $ew group-update \
+    --json "${updated_group}"
+
+  [ "${output}" = "${fixture}" ]
+}
+
+
+
```

Comment:
my only nitpic is too many new lines here (is there an autoformatter that can be used I wonder?)

Reply:
I adjusted the formatting.  No formatter readily available for bats files in VSC.